### PR TITLE
Allow embed timestamp to be Date in typings

### PR DESCRIPTION
**Pull request by oathompsonjones**
Type should be `string | Date` in order to allow for doing `timestamp: new Date()`

### CODE REVIEW

**Review by Skillz4Killz — DISMISSED** [review]

**Review by zneix — DISMISSED** [review]

**Review by Skillz4Killz — DISMISSED** [review]

**Comment by Skillz4Killz** [comment]
I think this PR is fine, but fair warning this could be considered a breaking change. Since it was accepting `string` before, it would not accept the `Date` option so most devs probably opted to do something like what I have

`new Date(time).toISOString()`

Im pretty sure this would break on compile time because you cant set a string to a `Date`. Will need testing when im able to do so

**Comment by Skillz4Killz** [comment]
Please, can we consider rewinding this back to the original commit of `Date | string`? I am certain I am not the only dev who was probably sending strings instead of Date. Here is how many errors I got from this change. 

image

I believe the best type would be `Date | string`

File: index.d.ts [hunk 1]
```diff
@@ -316,7 +316,7 @@ declare namespace Eris {
     title?: string;
     description?: string;
     url?: string;
-    timestamp?: string;
+    timestamp?: string | Date;
```
Comment (Skillz4Killz) [inline index.d.ts]:
```suggestion
    timestamp?: Date | string;
```

**Review by Skillz4Killz — DISMISSED** [review]
It's perfect. Slight modification to `Date | string` to have a Strict => Loose typings format would be nicer on the eyes imo.

**Comment by DonovanDMC** [comment]
I have always used `new Date().toISOString()`, making it take only a `Date` object would break literally every single one of my projects, thousands of files total (yes, I see that it has been changed back)

**Review by Skillz4Killz — APPROVED** [review]

**Review by DonovanDMC — APPROVED** [review]

**Review by abalabahaha — APPROVED** [review]

**Pull request merged**